### PR TITLE
Correction of the issue #146

### LINKED ISSUE #146: [New installation] Error when executing outside of a prestashop environment
Hello, 

The program crashs in both phar and php mode if no prestashop location is found at the current path.
The expected behaviour is to allow only install commands.

## Error content :
`There are no commands defined in the "install" namespace. `

### PR DESCRIPTION
**Pull request by m3uusi**
Correction of the issue #146

The script can now correctly be executed from directories where no prestashop is installed in phar mode to install one.
In php mode, the script can only be executed from the directory where the console script is located :
This is necessary because prestashop path is searched with relative paths (../config/...)

This P.R. also contains the correction of the #142 because the code was required for this correction to be tested.

### CODE REVIEW

**Comment by nenes25** [comment]
@m3uusi  I have issue on the merge of the branch can you rebase your modifications on current dev branch ?

File: src/Hhennes/PrestashopConsole/Command/Module/ListCommand.php [hunk 1]
```diff
@@ -35,35 +35,35 @@
  */
 class ListCommand extends Command
```
Comment (nenes25) [inline src/Hhennes/PrestashopConsole/Command/Module/ListCommand.php]: This command shoud not be in this pull request

**Review by nenes25 — CHANGES_REQUESTED** [review]
Hi can you rebase on current dev branch ? I have some conflicts

**Comment by m3uusi** [comment]
Branch merged with the current dev branch to resolve conflicts.

**Review by nenes25 — APPROVED** [review]
Sounds great thanks !

**Pull request merged**